### PR TITLE
boards: same70: Enable SPI0 on dts

### PR DESCRIPTION
**Pull request by qianfan-Zhao**
SAME70 XPLD board connnected spi0 to headers EXT1 and EXT2, so lets
enable SPI0 support on dts.

Signed-off-by: qianfan Zhao <qianfanguijin@163.com>

### CODE REVIEW

**Review by galak — CHANGES_REQUESTED** [review]
Can you add 'spi' to boards/arm/sam_e70_xplained/sam_e70_xplained.yaml under 'supported:'

File: boards/arm/sam_e70_xplained/sam_e70_xplained.dts [hunk 1]
```diff
@@ -65,6 +65,10 @@
 	status = "ok";
 };
 
+&spi0 {
+	status = "ok";
```
Comment (galak) [inline boards/arm/sam_e70_xplained/sam_e70_xplained.dts]: How are the chipselects wired on the board?  Using gpio? if so we should describe them here.

Comment (qianfan-Zhao) [reply boards/arm/sam_e70_xplained/sam_e70_xplained.dts]: There are nothing on spi bus, so I only enable spi0 and doesn't add any other nodes.

Besides: How to detect using gpio cs or using hardware cs control in a spi driver?

**Comment by qianfan-Zhao** [comment]
@galak 
> /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/drivers/spi/spi_sam.c:247:6: error: assignment discards ‘const’ qualifier from pointer target type [-Werror=discarded-qualifiers]
   tx = tx_bufs->buffers;
      ^
/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/drivers/spi/spi_sam.c:252:6: error: assignment discards ‘const’ qualifier from pointer target type [-Werror=discarded-qualifiers]
   rx = rx_bufs->buffers;
      ^
cc1: all warnings being treated as errors

This issue was fixed at pr #10620

**Pull request closed without merging**